usaspending_api/search/migrations/0008_awardsearch_table.py

```diff
@@ -115,7 +119,7 @@ class Migration(migrations.Migration):
         # Trick Django into believing this is a foreign primary key for purposes of using the ORM,
         # but avoid the headache that comes with foreign keys and the primary key constraint
         migrations.RunSQL(
-            sql="ALTER TABLE rpt.award_search DROP COLUMN id",
+            sql=";".join(additional_sql),
             reverse_sql="",
```

Comment:
Can you think of anytime where not having reverse_sql specified could be an issue? Think it would need to be added to allow this migration to be reversed (should that need ever come up).

Reply:
Unapplying migrations runs `DROP TABLE "rpt"."award_search" CASCADE;` so can disregard.